efficientdet/keras/utils_keras.py

```diff
@@ -45,7 +45,11 @@ def build_batch_norm(is_training_bn: bool,
     A normalized `Tensor` with the same `data_format`.
   """
   axis = 1 if data_format == 'channels_first' else -1
-  batch_norm_class = utils.batch_norm_class(is_training_bn, strategy)
+  if is_training_bn:
+    batch_norm_class = tf.keras.layers.experimental.SyncBatchNormalization
```

Comment:
Does SyncBatchNormalization work well on GPUs?

Could you add the following two lines:

```
# TODO(tanmingxing): compare them on TPU. 
# batch_norm_class = utils.batch_norm_class(is_training_bn, strategy)
```

Reply:
I met the same [issue](https://github.com/tensorflow/tensorflow/issues/39569) before.
Could you test batch renormalization which appears better with small batch size?